### PR TITLE
Update browserAction to action for MV3 docs

### PR DESCRIPTION
**Pull request by tomasdev**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by dotproto** [comment]
Thanks for the suggestion. This is a good change, but it only covers half of the API update. We also need to update `manifest.json` to move from `browser_action` to `action`. Could you update line 46 as well?

**Review by dotproto — CHANGES_REQUESTED** [review]
Thanks for the suggestion. This is a good change, but it only covers half of the API update. We also need to update `manifest.json` to move from `browser_action` to `action`. Could you update line 46 as well?

**Comment by tomasdev** [comment]
@dotproto done!

File: site/en/docs/extensions/mv3/manifest/activeTab/index.md [hunk 1]
```diff
@@ -46,13 +46,13 @@ See the [Page Redder][2] sample extension:
   "browser_action": {
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/manifest/activeTab/index.md:46]:
```suggestion
  "action": {
```

`browser_action` has been replaced by `action` in MV3.

**Review by dotproto — CHANGES_REQUESTED** [review]
Good catch, manifest_version also needed to be updated. One more change and I think we're good.

File: site/en/docs/extensions/mv3/manifest/activeTab/index.md (hunk 1 above)
Comment (tomasdev) [reply site/en/docs/extensions/mv3/manifest/activeTab/index.md]: Oops!

**Review by dotproto — APPROVED** [review]
LGTM, thanks again!

**Pull request closed without merging**